src/gui/src/browserWidget.cpp

```diff
@@ -313,7 +313,7 @@ void BrowserWidget::markModelModified()
 
 void BrowserWidget::updateModel()
 {
-  if (!model_modified_) {
+  if (!model_modified_ && model_->rowCount() != 0) {
```

Comment:
When does it happen that the model is modified but there are no rows?

Reply:
When hideEvent occurs  (which happens when you press the detach button), the model is cleared (but the model is not modified), and now the model will not be rebuilt on the show event resulting in the empty view.

Reply:
Why does detaching clear the model?

Reply:
I had added that to save memory on large designs, so I can turn off the call backs and not worry about that.
I think a data item model for this would be better in the long run, but that will take a little time to rework.